### PR TITLE
Blogpost deltatouch

### PR DESCRIPTION
**Pull request by lk108**
I just released the app in the OpenStore, so after review and corresponding updates, the post is ready for publishing.

### CODE REVIEW

File: _posts/2023-07-02-deltatouch.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+System notifications are possible, but require the user to disable background suspension and have the app running in the background. I haven't tested methodically how the latter affects battery life, but my first impressions are that it doesn't drain the battery very much.
+
+Things still missing include:
+* HTML emails
+* Webxdc
+* In-app scanning of QR codes (scanning with external app, copy to clipboard and pasting into DeltaTouch works)
+* Act as primary device to add second device
+* Encrypt database (may be important for UT as it currently doesn't offer encryption on OS level)
+* Connectivity status
+* "Last seen" green dot
```
Comment (adbenitez) [inline _posts/2023-07-02-deltatouch.md]:
```suggestion
* "Seen recently" green dot
```

**Review by adbenitez — APPROVED** [review]
nice!

File: _posts/2023-07-02-deltatouch.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+title: Introducing DeltaTouch, a new community created client for Ubuntu Touch
+author: lk108
+image: ../assets/blog/2023-07-deltatouch.png
+com_id: 110621944095573330
+---
+
+Hi, this is Lothar aka lk108. I'm today's guest on the blog and it's my pleasure to introduce DeltaTouch, a third-party client based on deltachat-core for the mobile phone OS [Ubuntu Touch](https://ubports.com) (UT). The app has just been [released on the OpenStore](https://open-store.io/app/deltatouch.lotharketterer) for both the xenial and the focal channel of UT. Here's how it looks like:
```
Comment (ralphtheninja) [inline _posts/2023-07-02-deltatouch.md]:
```suggestion
Hi, this is Lothar aka lk108. I'm today's guest on the blog and it's my pleasure to introduce DeltaTouch, a third-party client based on deltachat-core for the mobile phone OS [Ubuntu Touch](https://ubports.com) (UT). The app has just been [released on the OpenStore](https://open-store.io/app/deltatouch.lotharketterer) for both the xenial and the focal channel of UT. Here's what it looks like:
```

**Review by ralphtheninja — APPROVED** [review]
Very nicely written!

File: _posts/2023-07-02-deltatouch.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+image: ../assets/blog/2023-07-deltatouch.png
+---
+
+Hi, this is Lothar aka lk108. I'm today's guest on the blog and it's my pleasure to introduce DeltaTouch, a third-party client based on deltachat-core for the mobile phone OS [Ubuntu Touch](https://ubports.com) (UT). The app has just been [released on the OpenStore](https://open-store.io/app/deltatouch.lotharketterer) for both the xenial and the focal channel of UT. Here's what it looks like:
+
+<img src="../assets/blog/2023-07-chatlist.png" style="width:230px;" alt="" />
+<img src="../assets/blog/2023-07-surudark.png" style="width:230px;" alt="" />
+<img src="../assets/blog/2023-07-ambiance.png" style="width:230px;" alt="" />
+
+# Features
```
Comment (r10s) [inline _posts/2023-07-02-deltatouch.md]:
```suggestion
## Features
```

see below for comment

File: _posts/2023-07-02-deltatouch.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+* Encrypt database (may be important for UT as it currently doesn't offer encryption on OS level)
+* Connectivity status
+* "Seen recently" green dot
+* Clear chat
+
+These are planned to be implemented in the coming months.
+
+# Background and History
+
+## Why this app?
```
Comment (r10s) [inline _posts/2023-07-02-deltatouch.md]:
```suggestion
## Background and History

### Why this app?
```

File: _posts/2023-07-02-deltatouch.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
+I don't have to tell you which messenger is the best, right? Unfortunately, there was no app for the only smartphone OS I have ever used. Sure, it might be possible to run the Android app via waydroid, but waydroid massively impacts battery life, blocks GPS reads for the rest of the system and for some devices, it's not available at all. Therefore, a native app is the way to go.
+
+However, for a long time I felt that I wasn't competent to develop apps for UT, let alone something like a DC client. Granted, although it wasn't related to my main studies at all, many years ago I was able to attend an add-on computer science program at university, and I had some basic C++ skills. But I had never done anything with Qt/QML (the preferred framework for UT), and I didn't know anything about CMake or [clickable](https://clickable-ut.dev/en/latest/index.html) (the tool to build click packages as used by UT).
+
+## How it came into being
```
Comment (r10s) [inline _posts/2023-07-02-deltatouch.md]:
```suggestion
### How it came into being
```

File: _posts/2023-07-02-deltatouch.md [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+
+A very important aspect was that even when I didn't know how to handle a certain topic, I could always count on the Delta Chat and the UT AppDev community. Shout out to [Simon (treefit)](https://fosstodon.org/@treefit), link2xt and [adbenitez](https://mastodon.social/@adbenitez) from Delta and Maciek, dobey and Jonatan from the UT community for their friendly and competent help, very much appreciated! Thanks also to [Marko](https://kanoa.de/@playforvoices) for testing and valuable suggestions.
+
+To all of you out there who are in the same spot as I was two years ago: If you want to get involved into development, but think you are not capable of it - yes, it takes some learning and dedication, but it's absolutely doable. Don't hesitate to ask for help, the community is very willing to respond to your questions.
+
+# Contributing
```
Comment (r10s) [inline _posts/2023-07-02-deltatouch.md]:
```suggestion
## Contributing
```

File: _posts/2023-07-02-deltatouch.md [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+
+PRs are welcome at [https://codeberg.org/lk108/deltatouch](https://codeberg.org/lk108/deltatouch). If you're not a programmer, you can still contribute bug reports on Codeberg. You can also give feedback regarding usability, layout etc. Feel free to contact me in English or German via chat or classic email - username is deltatouch, domain is mailbox followed by org.
+
+# Download
```
Comment (r10s) [inline _posts/2023-07-02-deltatouch.md]:
```suggestion
## Download
```

**Review by r10s — APPROVED** [review]
thanks a lot for the app and for the nice blogpost (nb: i am also very happy that the "getting stated" section of the c-ffi was helpful and encouraging)

as discussed somewhere else, i would also like to add the ubuntu-touch app to the download page, but that better goes to another pr. EDIT: created https://github.com/deltachat/deltachat-pages/pull/693 for that

File: _posts/2023-07-02-deltatouch.md (hunk 1 above)
Comment (lk108) [reply _posts/2023-07-02-deltatouch.md]: I'm sorry, I just saw now that I could have clicked on "commit suggestion" which would have listed you as co-author. Instead, I have edited and committed from my local repo.

File: _posts/2023-07-02-deltatouch.md (hunk 2 above)
Comment (lk108) [reply _posts/2023-07-02-deltatouch.md]: Same as for the suggestion by adb above: I'm sorry, I just saw now that I could have clicked on "commit suggestion" which would have listed you as co-author. Instead, I have edited and committed from my local repo.

File: _posts/2023-07-02-deltatouch.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
+---
+title: Introducing DeltaTouch, a new community created client for Ubuntu Touch
+author: lk108
+image: ../assets/blog/2023-07-deltatouch.png
+---
+
+Hi, this is Lothar aka lk108. I'm today's guest on the blog and it's my pleasure to introduce DeltaTouch, a third-party client based on deltachat-core for the mobile phone OS [Ubuntu Touch](https://ubports.com) (UT). The app has just been [released on the OpenStore](https://open-store.io/app/deltatouch.lotharketterer) for both the xenial and the focal channel of UT. Here's what it looks like:
```
Comment (hpk42) [inline _posts/2023-07-02-deltatouch.md]:
```suggestion
Hi, this is Lothar aka [lk108](https://social.tchncs.de/@lk108). I'm today's guest on the blog and it's my pleasure to introduce DeltaTouch, a third-party client based on deltachat-core for the mobile phone OS [Ubuntu Touch](https://ubports.com) (UT). The app has just been [released on the OpenStore](https://open-store.io/app/deltatouch.lotharketterer) for both the xenial and the focal channel of UT. Here's what it looks like:
```

**Pull request merged**